### DISCUSSION TITLE
towards python as a "meta" kernel

### DISCUSSION THREAD

**Discussion by majidaldo** [marimo-team/marimo#2151] [Ideas]
Looks like this project has the right wirings and abstractions to make a truly polyglot notebook possible.

for data scientists, the big interop is between R and Python. just to start a discussion, would this be possible?

**Comment by akshayka** [comment]
Maybe possible, but certainly a very big project, especially if the R code is to be reactive. It's probably not something that we'll prioritize soon.

marimo is sort of polyglot in that it supports both Python and SQL (and markdown too I guess!), with some reactivity across the languages. Maybe that's what you meant as a "meta" kernel. Interesting way to think about it!
